### DISCUSSION TITLE
Frage zu Assigment 2

### DISCUSSION THREAD

**Discussion by MehmetCagferoglu** [irgroup-classrooms/dis08-2021#63] [Q&A]
#### Hallo Zusammen, wir hätten einige Fragen:

2. Make your program extract a full list of all genre names that are listed in column "listed_in". Make sure to extract only the genres and clean the data from non-letter characters if necessary. Save as a list. 
In der Spalte "listed_in" gibt es verschiedene Filme mit mehreren Genres. 
Sollen wir in der extrahierten Liste jeweils nur ein Genre oder alle angeben? <br>

*Beispiel 1:* 

- Biographisches
- Sport; Drama, Historisch
Oder 
*Beispiel 2:*
- Biographisches
- Sport
- Drama
- Historisch

3. Extend your program to count the distinct (unique) genre names.
Sollen wir die mehrere Genre Filme wie mehrere zählen oder diese als einzeln zählen?

4. Next, find how many entries in the disney plus catalog belong to each of the genres. Save the genres and corresponding counts in a dictionary.
Sollen wir die Filme mit mehreren Genres wie einen Punkt für jedes Genre zählen?

Vielen Dank und ein guten Rutsch!

**Comment by HaakFab** [comment]
Guten Morgen und ein frohes neues Jahr!
Behandeln Sie bitte "Drama, Historisch" als Drama und Historisch, also zu beiden Genres zugehörig.
Ich hoffe, dass das Ihre Frage beantwortet.

Viele Grüße,
Fabian Haak
